### PR TITLE
Project-pizza-AlexandraM

### PR DESCRIPTION
**Pull request by Cocofnas**

### CODE REVIEW

**Comment by Cocofnas** [comment]
https://comfy-bombolone-1e5fac.netlify.app/
https://codesandbox.io/s/variables-datatypes-forked-7kqm4m?file=/src/index.js

File: code/script.js [hunk 1]
```diff
@@ -1,19 +1,110 @@
...
+  alert(`You chose ${selectedFood} and ${subtypeName[parseInt(subtype) - 1]} as your meal.`);
+} else if (selectedFood === "Salad") {
+  alert(`You chose ${selectedFood} and ${subtypeName[parseInt(subtype) - 1]} as your meal.`);
+}
+// Step 4 - Age
+// Your code goes here
+
+let age = prompt ("Type your age in numbers")
+if (age>=18) {
+  alert (`The price is 150 SEK`);
```
Comment (Cocofnas) [inline code/script.js:78]: `The price for an adult pizza is 150 SEK`

File: code/script.js [hunk 2]
```diff
@@ -1,19 +1,110 @@
...
+}
+else if (age<=18) {
+  alert (`The price is 100 SEK`);
```
Comment (Cocofnas) [inline code/script.js:81]: `The price for a child size pizza is 100 SEK

File: code/script.js [hunk 3]
```diff
@@ -1,19 +1,110 @@
...
+else if (selectedFood === "Salad") {
+alert(
+  `You chose ${selectedFood} and ${subtypeName[parseInt(subtype) - 1]} as your meal.`);
+}
+
+let order = prompt ("Do you want to place an order? Yes or no")
+if (order==="yes") {
+  alert (`Thank you for your order.`);
+}
+else if (order==="no") {
```
Comment (Cocofnas) [inline code/script.js:108]: ´remove if

File: code/script.js [hunk 4]
```diff
@@ -1,19 +1,110 @@
...
+// Step 5 - Order confirmation
+// Your code goes here
+
+if (selectedFood === "Pizza")
+{
+  alert(
+  `You chose ${selectedFood} and ${subtypeName[parseInt(subtype) - 1]} as your meal.`);
+  
+
+  } else if (selectedFood === "Pasta") {
```
Comment (Cocofnas) [inline code/script.js:94]: remove if

**Review by Shiqwan — APPROVED** [review]
In general, great effort,  you get the work done, my tummy is growling and I'm wondering what an Elf pasta is?? The code rolls along and it feels clear and to the point!

**Pull request closed without merging**